### DISCUSSION TITLE
Can someone remind me of the URL to see the annotated images please :-)

### DISCUSSION THREAD

**Discussion by schford** [neilenns/node-deepstackai-trigger#421] [Q&A] [answered]
I cant seem to find the URLs for looking at the processed images, I found it before but seem to have misplaced it and am setting up a new system and just want to check it is all working.

Cheers

Stuart

**Comment by neilenns** [comment, marked as answer]
Looks like this: http://localhost:4242/annotations/

**Comment by hiro5id** [comment]
I forgot too :) but it is easy to find in the source code:
https://github.com/danecreekphotography/node-deepstackai-trigger/blob/344db382d0df097d952db381f5db24d69680737e/src/WebServer.ts#L27
